core/electron/package.json

```diff
@@ -39,7 +52,7 @@
     "@itwin/core-bentley": "workspace:^4.10.0-dev.30",
     "@itwin/core-common": "workspace:^4.10.0-dev.30",
     "@itwin/core-frontend": "workspace:^4.10.0-dev.30",
-    "electron": ">=23.0.0 <34.0.0"
+    "electron": ">=28.0.0 <34.0.0"
```

Comment:
what happens if we leave the min ver as is?

Reply:
Electron "properly" supports ESM from version 28. If we're delivering both CJS and ESM, we could leave version 23 as minimum, I think.